pootle/apps/pootle_app/views/admin/dashboard.py

```diff
@@ -197,33 +197,25 @@ def optimal_depcheck():
         optimal.append({'dependency': 'db', 'text': text})
 
     if depcheck.test_cache():
-        if depcheck.test_memcache():
-            if not depcheck.test_memcached():
-                # memcached configured but connection failing
-                optimal.append({
-                    'dependency': 'cache',
-                    'text': _("Pootle is configured to use memcached as a "
-                              "caching backend, but can't connect to the "
-                              "memcached server. Caching is currently "
-                              "disabled.")
-                })
-            else:
-                if not depcheck.test_session():
-                    text = _("For optimal performance, use django.contrib."
-                             "sessions.backends.cached_db as the session "
-                             "engine.")
-                    optimal.append({'dependency': 'session', 'text': text})
-        else:
+        if not depcheck.test_cache_server_connection():
+            # Server configured but connection failing
             optimal.append({
                 'dependency': 'cache',
-                'text': _("Pootle is configured to use memcached as caching "
-                          "backend, but Python support for memcached is not "
-                          "installed. Caching is currently disabled.")
```

Comment:
@julen @unho I'll fix these apart as their own commit otherwise there is just too much noise.